storage/backup/backup-cli/src/lib.rs

```diff
@@ -17,8 +18,37 @@ use storage_client::{StorageRead, StorageReadServiceClient};
 
 pub type FileHandle = String;
 
+pub struct BackupService {
```

Comment:
Is this a service?

Reply:
it represents the service on the client side. But anyway I'm changing it to `BackupServiceClient`